webapp/src/components/news-form/NewsForm.test.js

```diff
@@ -0,0 +1,61 @@
+import {describe, expect, it} from "@jest/globals";
+import {mount} from "@vue/test-utils";
+import NewsForm from "@/components/news-form/NewsForm";
+
+require('regenerator-runtime/runtime');
+
+const itemTitle = "Fake News";
+const itemTitleTooLong = "This Text is longer than the accepted sixtyfour characters!";
+
+describe('NewsForm', () => {
+	
+	/*
+	TODO: 
+	- understand why 'valid' does not change value when setData() is executed
+	- afterwards refactor the test
+	*/
+	
+	it('should accept less than 64 character', async () => {
+
+	const wrapper = mount(NewsForm, {
+        data() {
+            return {
+				newsTitle: ''
+            }
+        }
+    });
+      
+    await wrapper.setData({
+		valid: true,
+        newsTitle: itemTitle
+    });
+	
+	//await wrapper.vm.$nextTick();
+      
+	expect(wrapper.vm.newsTitle).toBe(itemTitle);
```

Comment:
Don't test internal properties. Instead check what you can see:
```js
expect(wrapper.find('input').attributes('value')).toEqual('Fake News')
```
Also use hard-coded assertion values to avoid silently passing false-negative test cases.